Library/Homebrew/requirements/ruby_requirement.rb

```diff
@@ -33,4 +31,34 @@ def display_s
       name
     end
   end
+
+  private
+
+  def suitable_ruby
+    rubies.detect { |ruby| suitable?(ruby) }
+  end
+
+  def rubies
+    rubies = which_all("ruby")
+    if ruby_formula && ruby_formula.installed?
```

Comment:
Turn `ruby_formula` into a variable and just use `Formula["ruby"]`; you can assume it'll never throw a `FormulaUnavailableError` exception.